frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -0,0 +1,118 @@
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { browser, ExpectedConditions as until, $ } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+import { isNodeReady } from '@console/shared/src/selectors/node';
+import {
+  clickKebabAction,
+  goToInstalledOperators,
+  ocsOp,
+  storageClusterRow,
+  storageClusterView,
+} from '../views/add-capacity.view';
+import { EXPANDWAIT, KIND, NS, POD_NAME_PATTERNS, SECOND } from '../utils/consts';
+import {
+  getPodPhase,
+  getPodRestartCount,
+  isPodPresent,
+  testPodIsRunning,
+  testPodIsSucceeded,
+} from '../utils/helpers';
+
+const storageCluster = JSON.parse(execSync(`kubectl get -o json -n ${NS} ${KIND}`).toString());
+
+let clusterJSON;
+let previousCnt;
+let previousPods;
+let updatedClusterJSON;
+let updatedCnt;
+let updatedPods;
+
+describe('Check add capacity functionality for ocs service', () => {
+  beforeAll(async () => {
+    const { name } = storageCluster.items[0].metadata;
+    clusterJSON = JSON.parse(execSync(`kubectl get -o json -n ${NS} ${KIND} ${name}`).toString());
+    previousCnt = _.get(clusterJSON, 'spec.storageDeviceSets[0].count', undefined);
+    const uid = _.get(clusterJSON, 'metadata.uid', undefined).toString();
+    previousPods = JSON.parse(execSync(`kubectl get pods -n ${NS} -o json`).toString());
+
+    await goToInstalledOperators();
+    await click(ocsOp);
+    await click(storageClusterView);
+
+    await clickKebabAction(uid, 'Add Capacity');
```

Comment:
What happens after clicking the Kebab?

Reply:
I see that you have moved `verifyFields` to view. It is better to keep it here for clarity.
```
await clickKebabAction(uid, 'Add Capacity');
await verifyFields();
```
Looks much clearer to me. 
Also, better to add `expect` to scenario rather than in view.

Reply:
@umangachapagain the intention to add expect here to make the function generic enough, so that even if we want to use kebab actions, we should be able to do that.

Reply:
But, when I look here, it is not obvious to me what happens next. I'd prefer readability over generalization.